src/core/pages/AccountInfo.vue

```diff
@@ -28,6 +32,23 @@
       </div>
     </div>
 
+    <!-- New password modal -->
+    <ni-modal v-model="newPasswordModale" @hide="resetForm">
+      <template slot="title">
+        Modifier mon <span class="text-weight-bold">mot de passe</span>
+      </template>
+      <ni-input in-modal v-model.trim="mergedUserProfile.local.password" type="password"
+        caption="Nouveau mot de passe" :error-label="passwordError($v.mergedUserProfile.local.password)" required-field
+        @blur="$v.mergedUserProfile.local.password.$touch" :error="$v.mergedUserProfile.local.password.$error" />
+      <ni-input in-modal v-model.trim="passwordConfirm" :error="$v.passwordConfirm.$error" type="password"
+        caption="Confirmation mot de passe" :error-label="passwordConfirmError" @blur="$v.passwordConfirm.$touch"
+        required-field />
+      <template slot="footer">
+        <q-btn no-caps class="full-width modal-btn" label="Modifier" :loading="loadingPasswordChange" color="primary"
+           icon-right="done" :disable="$v.$invalid || passwordConfirm === ''" @click="submitPasswordChange" />
```

Comment:
Je pense que tu peux mettre le password de l'utilisateur en required. Comme ca tu peux verifier que les deux mdp sont valides et ca devrait gerer le cas ou il est vide.

Reply:
non ça gere pas le cas où c'est vide, car la première condition de updateUser (userMixin) `if (this.tmpInput === get(this.mergedUserProfile, path))` est verifié et donc on return rien.

A cause de :
- ça,
-  plus des gestions des pop dans les updateAlenviUser et des if partout dans le updateAlenviUser où je travail
- plus de tmpInput qui servent a rien dans la modale

>> j'ai fait un fonction specifique pour le update du mdp